src/compiler/utilities.ts

```diff
@@ -850,10 +850,88 @@ namespace ts {
         return node && (node.kind === SyntaxKind.GetAccessor || node.kind === SyntaxKind.SetAccessor);
     }
 
-    export function isClassLike(node: Node): node is ClassLikeDeclaration {
+    export function isClassLike(node: Node | undefined): node is ClassLikeDeclaration {
         return node && (node.kind === SyntaxKind.ClassDeclaration || node.kind === SyntaxKind.ClassExpression);
     }
 
+    export function isImmediatelyInvokedFunctionExpression(node: Node) {

```

Comment:
No need to export. `isTypeScriptClassWrapper` could also be moved closer to its use.